fhir-bucket/src/main/java/com/ibm/fhir/bucket/api/IResourceEntryProcessor.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.bucket.api;
+
+
+/**
+ * Process a ResourceEntry
```

Comment:
from where? what is the difference between the processor and the resource handler? fhir-bucket/src/main/java/com/ibm/fhir/bucket/api/IResourceHandler.java

Reply:
Converged over iterations. Probably could be combined, but I don't think a necessary change at this stage.